golint/golint.go

```diff
@@ -11,15 +11,23 @@ import (
 	"flag"
 	"fmt"
 	"go/build"
-	"io/ioutil"
+	"go/parser"
 	"os"
 	"path/filepath"
+	"sort"
 	"strings"
 
+	"golang.org/x/tools/go/loader"
+
 	"github.com/golang/lint"
 )
 
 var minConfidence = flag.Float64("min_confidence", 0.8, "minimum confidence of a problem to print it")
+var buildTags stringsFlag
+
+func init() {
+	flag.Var(&buildTags, "tags", "a list of build tags to consider satisfied during the build")
```

Comment:
If a file has a build tags it cannot be used by other files until the flag conditions are met. So I think it is fine, govet does the same and that is part of the official Go. Btw why isn't golint official?!